aopy/analysis/base.py

```diff
@@ -2231,4 +2231,53 @@ def windowed_xval_lda_wrapper(data, labels, samplerate, lags=3, nfolds=5, regula
     elif return_confusion_matrix:
         return decoding_accuracy, time_axis, cm
     else:
-        return decoding_accuracy, time_axis
\ No newline at end of file
+        return decoding_accuracy, time_axis
+    
+def simulate_trajectories(targets, origin=[0.0, 0.0, 0.0], resolution=100):
+    """
+    Simulates reach trajectories from a given origin to a list of target points in 3D space.
+    The function calculates the trajectories as straight lines from the origin to each target.
+
+    Args:
+        targets (numpy.ndarray or list of lists/tuples): A list or array of target points
+            in 3D space (each entry is a 3D coordinate).
+        origin (list or tuple, optional): The origin point from which the trajectories
+            are simulated. Default is [0.0, 0.0, 0.0].
+        resolution (int, optional): The number of points used to render each trajectory.
+            Default is 100.
+
+    Returns:
+        numpy.ndarray: An array of simulated trajectories, each being a series of points
```

Comment:
could you specify the size of the array? something like this?
```suggestion
        (ntargets, resolution, ndim) array: An array of simulated trajectories, each being a series of points
```